src/scripts/main.js

```diff
@@ -1,3 +1,175 @@
+
+    listOrder = !listOrder;
+
+    sorted.forEach(row => {
+      tableBody.appendChild(row);
+    });
+  }
+
+  if (evt.target.tagName === 'TD') {
+    evt.stopPropagation();
+    tablerows.forEach(rowClass => rowClass.classList.remove('active'));
+    evt.target.parentNode.classList = 'active';
+    evt.target.contentEditable = true;
+  }
+});
+
+// form
+
+const form = document.createElement('form');
+
+form.classList.add(`new-employee-form`);
+
+for (const value in formFields) {
+  const newEntry = document.createElement('label');
+
+  newEntry.textContent = capitalise(value);
+  newEntry.for = value;
+  form.appendChild(newEntry);
+
+  const newInput = document.createElement('input');
+
+  newInput.setAttribute('data-qa', value);
+  newInput.type = formFields[value];
+  newInput.name = value;
+  newEntry.appendChild(newInput);
+
+  if (formFields[value] === 'select') {
+    newInput.remove();
+
+    const officeSelect = document.createElement('select');
+
+    officeSelect.name = 'office';
+    officeSelect.setAttribute('data-qa', value);
+
+    offices.forEach(office => {
+      const newOffice = document.createElement('option');
+
+      newOffice.value = office;
+      newOffice.textContent = office;
+      officeSelect.appendChild(newOffice);
+    });
+
+    newEntry.appendChild(officeSelect);
+    form.appendChild(newEntry);
+  }
+}
+
+const button = document.createElement('button');
+
+button.textContent = 'Save to table';
+button.type = 'submit';
+button.classList.add('button__select');
+
+form.appendChild(button);
+
+table.insertAdjacentElement('afterend', form);
+
+const formFilled = document.querySelector('.new-employee-form');
+
+formFilled.addEventListener('submit', function(evt) {
+  evt.preventDefault();
+
+  const formData = new FormData(evt.target);
+  const rowToInsert = document.createElement('tr');
+  const enteredName = formData.get('name');
+  const enteredAge = formData.get('age');
+  const enteredPosition = formData.get('position');
+
+  rowToInsert.setAttribute('data-qa', 'success');
+
+  const notification = document.createElement('div');
+  const notificationTitle = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.classList.add('notification');
+  notificationTitle.classList.add('title');
+
+  let message = 'All field are correct and the table will be updated';
+  let messageClass = 'success';
+
+  if (enteredName.length < 4) {
+    message = 'Text entries must be longer than four characters in length';
+    messageClass = 'error';
+  }
+
+  if (enteredPosition <= 0) {
+    message = 'Position must be entered';
+    messageClass = 'error';
+  }
+
+  if (enteredAge < 18 || enteredAge > 90) {
+    message = 'Age entry must be between 18 years and 90 years';
+    messageClass = 'error';
+  }
+
+  notificationTitle.textContent = message;
+  notification.classList.add(messageClass);
+  notification.append(notificationTitle);
+  form.append(notification);
```

Comment:
Don't append a notification inside your form, because it will look weird:
![image](https://github.com/mate-academy/js_employees_table_DOM/assets/25587299/e49d0995-4cc0-4dbf-94fc-c64e6c079482)

You can use `form.after(notification)` to render your notification as a form sibling but not a child.